ui/app/common/pnc-client/resources/helpers/buildStatusHelper.js

```diff
@@ -125,10 +125,11 @@
           }
 
           function isSuccess(build) {
-            return !isFailed(build);
+            var status = getStatus(build);
```

Comment:
@dominikbrazdil  please preferf `const` next time, or `let` if it needs to be reassigned

I think status `DONE` is not valid option anymore, so I would prefer using `$isSuccess()` in `pncBrewPushButton.js` or add comment why it can not be there

Reply:
Ok, if it is not valid anymore, then I'll leave there just SUCCESS state. Till now it returned true on every state that is not failed and that includes many of them. I just did not want to break something that may be using DONE status. Should I create an issue for removing the old statuses?

Reply:
+1 to create new issue to refactor `DONE` status out in favour of `SUCCESS`, there can be also some other deprecated statuses so creating a new jira fixing it all would be the best

Feel free to mention also this file https://github.com/project-ncl/pnc/blob/master/constants/src/main/java/org/jboss/pnc/enums/BuildStatus.java#L165 there.